components/did/Core/Src/DID.c

```diff
@@ -195,20 +204,20 @@ void update_DID_screen()
 			UpdateScreenTitles(PAGE_0);
 
 			// Get pointers to cyclic data variables
-			float* simulation_speed = GET_CYCLIC_DATA( data_simulation_speed );
+			float* vehicle_velocity = GET_CYCLIC_DATA( data_vehicle_velocity );
 			uint8_t* battery_soc = GET_CYCLIC_DATA( data_battery_SOC );
 			uint8_t* MCB_drive_state = GET_CYCLIC_DATA( data_MCB_drive_state );
-			float* target_velocity = GET_CYCLIC_DATA( data_target_velocity );
```

Comment:
Since we dont get cyclic data for the target_velocity, we should remove the Setters and initializer for it
![image](https://github.com/UBC-Solar/firmware_v3/assets/117491745/580a4fc4-1226-46bf-b9ae-df1ce19aecb4)
![image](https://github.com/UBC-Solar/firmware_v3/assets/117491745/e13db708-089d-4d47-b312-d1b29b441ae0)

Reply:
This was cruise. Bad naming. There is a separate simulation velocity/target still there.